paddle/fluid/operators/ngraph/ngraph_engine.cc

```diff
@@ -590,7 +594,7 @@ void NgraphEngine::Run(const framework::Scope& scope,
       }
       bool is_persistable =
           (p_persistables->find(vi) != p_persistables->end()) ? true : false;
-      if (is_test && is_persistable) {
+      if (!is_training && is_test && is_persistable) {
```

Comment:
`is_training` and `is_test` are duplicated?

Reply:
This can be simplified, but it will need changes otherwhere as well. I am trying to minimize change at this late release stage. That may involve more time for review and  validation.  Will follow up to remove duplicates after the release.